libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -1296,4 +1296,38 @@ int lua_gps_inject_data(lua_State *L)
 
 #endif  // AP_GPS_ENABLED
 
+#if !defined(HAL_BUILD_AP_PERIPH)
+int lua_AP_Vehicle_set_target_velocity_NED(lua_State *L)
```

Comment:
We discussed the addition of this Lua only function and I wasn't a fan.  Others including @tpwrules said he thought it was OK